.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "python.linting.enabled": true,
+  "python.linting.enabled": false,
```

Comment:
I have set this so that it enforces the code writer to adhere to the code style :) Please, change back to true.

Reply:
fixed.

Reply:
not fixed